app/helpers/application_helper.rb

```diff
@@ -1,2 +1,9 @@
 module ApplicationHelper
+  def form_group_tag(errors, &block)
+    if errors.any?
+      content_tag :div, capture(&block), class: 'form-group has-error'
+    else
+      content_tag :div, capture(&block), class: 'form-group'
+    end
+  end
```

Comment:
How could this method body be rewritten to remove the duplication between lines 4 and 6?